### DISCUSSION TITLE
Only one instance

### DISCUSSION THREAD

**Discussion by he-igel** [ChromiumDotNet/WebView2.DevTools.Dom#16] [Q&A] [answered]
The documentation says : Only a single WebView2DevToolsContext should exist at any given time.
Is this true for one webview2 instance or for the application which can use x webview2 instances?

Thank you!

**Comment by amaitland** [comment, marked as answer]
One `WebView2DevToolsContext` instance per [CoreWebView](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2?view=webview2-dotnet-1.0.1722.45) instance.
